locale/de.po

```diff
@@ -3887,141 +4014,163 @@ msgstr "Hochgeladene Datei wird finalisiert..."
-msgid "The project will sync in background while you work. You can check the sync status on the bottom right corner of Audacity at any time"
+msgid ""
+"The project will sync in background while you work. You can check the sync "
+"status on the bottom right corner of Audacity at any time"
 msgstr ""
+"Während Sie arbeiten, wird das Projekt im Hintergrund synchronisiert. Sie "
+"können den Synchronisierungsstatus jederzeit in der unteren rechten Ecke von"
+" Audacity überprüfen"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/SyncInBackroundDialog.cpp
 #: src/AudioPasteDialog.cpp
 msgid "Continue"
 msgstr "Fortsetzen"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/SyncSucceededDialog.cpp
-#, fuzzy
 msgid "Success!"
-msgstr "Erfolg"
+msgstr "Erfolg!"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/SyncSucceededDialog.cpp
-msgid "All saved changes will now update to the cloud. You can manage this file from your uploaded projects page on audio.com"
+msgid ""
+"All saved changes will now update to the cloud. You can manage this file "
+"from your uploaded projects page on audio.com"
 msgstr ""
+"Alle gespeicherten Änderungen werden nun in der Cloud aktualisiert. Sie "
+"können diese Datei über die Seite Ihrer hochgeladenen Projekte auf Audio.com"
+" verwalten"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/SyncSucceededDialog.cpp
-#, fuzzy
 msgid "View online"
-msgstr "Zeitleiste"
+msgstr "Online ansehen"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/SyncSucceededDialog.cpp
 msgid "Done"
-msgstr ""
+msgstr "Erledigt"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/UnsyncedProjectDialog.cpp
-#, fuzzy
 msgid "Cloud project incomplete"
-msgstr "Konnte Komponente nicht finden"
+msgstr "Cloud-Projekt unvollständig"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/UnsyncedProjectDialog.cpp
-msgid "The latest version of this project was not fully uploaded to audio.com. You can load the last complete version instead."
+msgid ""
+"The latest version of this project was not fully uploaded to audio.com. You "
+"can load the last complete version instead."
 msgstr ""
+"Die neueste Version dieses Projekts wurde nicht vollständig auf Audio.com "
+"hochgeladen. Sie können stattdessen die letzte vollständige Version laden."
 
 #: modules/mod-cloud-audiocom/ui/dialogs/UnsyncedProjectDialog.cpp
-#, fuzzy
 msgid "Load latest"
-msgstr "Dateien laden"
+msgstr "Neueste laden"
 
 #: modules/mod-cloud-audiocom/ui/dialogs/UnsyncedProjectDialog.cpp
-msgid "No version of this project has been fully uploaded to audio.com. It cannot be loaded."
+msgid ""
+"No version of this project has been fully uploaded to audio.com. It cannot "
+"be loaded."
 msgstr ""
+"Keine Version dieses Projekts wurde vollständig auf Audio.com hochgeladen. "
+"Es kann nicht geladen werden."
 
 #: modules/mod-cloud-audiocom/ui/dialogs/UploadCanceledDialog.cpp
-#, fuzzy
 msgid "You have canceled this upload to audio.com"
-msgstr "Hochladen zu Audio.com abbrechen"
+msgstr "Sie haben diesen Hochladen auf Audio.com abgebrochen"
```

Comment:
"Hochladen" is incorrect here. German just borrowed [Upload](https://www.duden.de/rechtschreibung/Upload).